### PR TITLE
main/perl-http-body: Update to 1.22

### PR DESCRIPTION
**Pull request by bratkartoffel**
https://metacpan.org/source/GETTY/HTTP-Body-1.22/Changes

### CODE REVIEW

File: main/perl-http-body/APKBUILD [hunk 1]
```diff
@@ -12,35 +12,28 @@ makedepends="perl-dev perl-test-deep"
...
 source="http://search.cpan.org/CPAN/authors/id/G/GE/GETTY/HTTP-Body-$pkgver.tar.gz
-	CVE-2013-4407.patch
 	"
 
-_builddir="$srcdir"/HTTP-Body-$pkgver
+builddir="$srcdir"/HTTP-Body-$pkgver
+
+# secfixes:
+#   1.22-r0:
+#     - CVE-2013-4407
```
Comment (ncopa) [inline main/perl-http-body/APKBUILD]: This secfixes comment is not correct. It indicates that the versions prior to 1.22-r0 is vulnerable to CVE-2013-4407, which is not true. We had a patch for it, which is removed with this commit.

The secfix comment needs to say which .apk version we fixed this CVE in alpine.

Comment (ncopa) [reply main/perl-http-body/APKBUILD]: use git log to find out exactly when we fixed this issue.

**Comment by ncopa** [comment]
CVE-2013-4407 was fixed with 213ebd008f5f44ea962bebeb139ec959c8c5ce4d, so the `1.17-r0` had the fix.

File: main/perl-http-body/APKBUILD (hunk 1 above)
Comment (bratkartoffel) [reply main/perl-http-body/APKBUILD]: Thank you, I just corrected that line

**Comment by bratkartoffel** [comment]
This has already been applied with 7454a74b607d2ea338d1742b0ae51257b74fafe9

**Pull request closed without merging**